src/utils/numAbbr.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @flow
+ * @file Function to abbreviate a number as a string in a locale-sensitive manner.
+ * @author Box
+ */
+
+import IntlMessageFormat from 'intl-messageformat';
+import { languages, numbers } from 'box-locale-data';
+
+interface NumAbbrOptions {
+    length: string;
+    locale: string;
+    numbersData: object;
+}
+
+function numAbbrWithLocale(numbersData: object, num: number, locale: string, options?: NumAbbrOptions): string {
+    if (!num) return '0';
+    let { length } = options || {};
+    length = length || 'short';
```

Comment:
I think 'short' and other allowed values should become an enum. See Tooltip.tsx or Button.tsx for examples.

Reply:
done